Ktisis/Interface/Windows/Workspace.cs

```diff
@@ -95,11 +95,16 @@ public class Workspace {
 				Coordinates();
 
 				ImGui.Separator();
-
-				var _ = false;
-				if (ImGui.Checkbox("Toggle Posing", ref _)) {
-					// TODO
+				
+				if (!Ktisis.IsInGPose)
```

Comment:
No need to disable the hooks every frame if they're not enabled.
```suggestion
				if (!Ktisis.IsInGPose && PoseHooks.PosingEnabled)
```